phpunit.xml.dist

```diff
@@ -34,6 +34,9 @@
                             <element>
                                 <string>eLife\Journal\Security</string>
                             </element>
+                            <element>
+                                <string>eLife\Journal\Security\Voter</string>
```

Comment:
This still uses the Symfony magic rather than a proper clock service.

Reply:
<img src="http://images.huffingtonpost.com/2016-03-07-1457387813-3802365-PHOTO2.jpeg" width=200 />